tasks/messages/message.go

```diff
@@ -42,8 +42,8 @@ func (p *Task) ProcessMessages(ctx context.Context, ts *types.TipSet, pts *types
 	defer span.End()
 
 	report := &visormodel.ProcessingReport{
-		Height:    int64(pts.Height()),
-		StateRoot: pts.ParentState().String(),
+		Height:    int64(ts.Height()),
```

Comment:
This is incorrect. The data for the message task is primarily from the parent tipset pts. The child tipset ts contains only the receipts. See https://github.com/filecoin-project/sentinel-visor/pull/650